src/obs-shaderfilter.c

```diff
@@ -262,6 +262,8 @@ static void shader_filter_reload_effect(struct shader_filter_data *filter)
 	if (filter->effect == NULL)
 	{
 		blog(LOG_WARNING, "[obs-shaderfilter] Unable to create effect. Errors returned from parser:\n%s", (errors == NULL || strlen(errors) == 0 ? "(None)" : errors));
+		bfree(errors);
```

Comment:
Should this bfree() only be done if errors is not NULL?

Reply:
Ah nevermind, I found the bfree() implementation in OBS and it does the NULL check internally.